helm/falco/templates/podlogs.yaml

```diff
@@ -0,0 +1,162 @@
+{{- if .Values.observability.enabled }}
+---
+# PodLogs for falco-exporter (all logs - non-sensitive metrics only)
+apiVersion: monitoring.grafana.com/v1alpha2
+kind: PodLogs
+metadata:
+  name: falco-exporter-logs
+  namespace: {{ .Release.Namespace }}
+  labels:
+    giantswarm.io/managed-by: security-team
+    giantswarm.io/service-type: managed
+    application.giantswarm.io/team: shield
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
We already have a helper that contains some of these:

See for example:
https://github.com/giantswarm/falco-app/blob/acb81648b1014cacaa0382417147204840ae30d4/helm/falco/templates/falco-ciliumnetworkpolicy.yaml#L8

and:
https://github.com/giantswarm/falco-app/blob/acb81648b1014cacaa0382417147204840ae30d4/helm/falco/templates/_helpers.tpl#L43:L56